Aquality.Selenium/src/Aquality.Selenium/Browsers/LocalBrowserFactory.cs

```diff
@@ -53,5 +58,20 @@ private Browser CreateBrowser()
             }
             return new Browser(driver, Configuration);
         }
+
+        private static void SetUpDriver(IDriverConfig driverConfig, IDriverSettings driverSettings)
+        {
+            var architecture = driverSettings.SystemArchitecture.Equals(Architecture.Auto) ? ArchitectureHelper.GetArchitecture() : driverSettings.SystemArchitecture;
```

Comment:
Can we just use ArchitectureHelper.GetArchitecture()?

Reply:
no we cannot. not working for me (I have x64 machine but need x32 webdriver)